docs/reference/sql/functions/date-time.asciidoc

```diff
@@ -91,8 +89,6 @@ include-tagged::{sql-specs}/docs.csv-spec[dtIntervalMul]
 
 ==== Functions
 
-beta[]
```

Comment:
This is the problem. You have two headings of the same level with no content between them. Something needs to go in this space here. When I put a comment here: `//comment`, the docs built for me.

Reply:
Cheers!